<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComSettings</class>
 <widget class="QDialog" name="ComSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>351</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configuration port</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DATA BITS</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxStopBits">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckBoxRts">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>70</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RTS</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckBoxDtr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>70</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DTR</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxParity">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxBaudrate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxDataBits">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BAUDRATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP BITS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PARITY</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
